### ISSUE TITLE
Memory leak in wui.Image

### ISSUE THREAD

**Issue by NeanderThaler** [gonutz/wui#21]
i think that there is a memory leak when creating images

to my current understanding:
w32.CreateDIBSection is called to allocate memory
when the image is deleted by GC, this memory is not freed (a call to w32.DeleteObject is missing)

can anyone confirm?

**Comment by gonutz** [comment]
You are right, the `Image.bitmap w32.HBITMAP` is allocated but there is currently no way in the API to free it. We might want to add a `Destroy()` function or something along those lines. Good catch! :-)

**Comment by gonutz** [comment]
I have not had time lately, but I actually do have some time on my hands right now, so I am aiming to fix this in the near future.

While we chat here, what do you all use `wui` for, I am very interested in other people's use cases - if you are willing to share :-D

**Comment by vault-thirteen** [comment]
I was doing some research about GUI in Golang and found your repository. I have also found out that at this moment of time there are no good libraries written in Go for GTK and Qt, so the only option for me to create a usable GUI was to use Windows API.
